cmd/sanitize-prow-jobs/main.go

```diff
@@ -15,27 +17,66 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
-const defaultCluster = "api.ci"
-
 type options struct {
 	prowJobConfigDir string
+	configPath       string
 
 	help bool
 }
 
+type Config struct {
+	Clusters struct {
+		Default      string `json:"default"`
+		Distribution map[string]struct {
+			Jobs   []string `json:"jobs"`
+			Files  []string `json:"files"`
+			Groups []string `json:"groups"`
+		} `json:"distribution"`
+		Groups map[string][]string       `json:"groups"`
+		REMap  map[string]*regexp.Regexp `json:"-"`
```

Comment:
This is the REMap for the files, right? Why is it not alongside of files but a level higher? Also please name `FileREMap`, otherwise we have to rename it should we ever need another `REMap`

Reply:
Renamed.

> Why is it not alongside of files but a level higher?
That was my first try too. I think the current structure is simpler.

Reply:
Done.